client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -117,6 +118,9 @@ export const ApplicationsTable: React.FC = () => {
     setIsEditAssessmentConfirmDialogOpen,
   ] = React.useState<Boolean>(false);
 
+  const [isEditReviewConfirmDialogOpen, setIsEditReviewConfirmDialogOpen] =
+    React.useState<Boolean>(false);
```

Comment:
Should be using lowercase 'boolean' here as the uppercase 'Boolean' indicates an object type whereas the lowercase is for the boolean primitive. Looks like this isn't breaking anything, though.